server/database/index.js

```diff
@@ -1,48 +1,16 @@
-// const mysql = require('mysql');
-
-// const connection = mysql.createConnection({
-//   host: 'localhost',
-//   // user: 'traveler',
-//   // password: 'travel',
-//   user: 'root',
-//   password: 'mypass',
-//   database: 'image_carousel',
-// });
-
-// connection.connect();
-
-// module.exports = connection;
-
 const { Pool, Client } = require('pg');
 
-const pool = new Pool({
+module.exports.pool = new Pool({
   user: 'images',
   host: 'localhost',
   database: 'image_carousel',
-  // password: 'images',
-  // port: 3007,
+  max: 250,
+  idleTimeoutMillis: 2000,
+  maxUses: 1000,
 });
 
-// pool.query('SELECT NOW()', (err, res) => {
-//   console.log(err, res);
-//   console.log(err);
-//   pool.end();
-// });
-
-const client = new Client({
+module.exports.client = new Client({
```

Comment:
You can probably just keep one db connection implementation rather than maintain both 'client' and pool'